examples/pointcloud_globe.js

```diff
@@ -0,0 +1,81 @@
+/* global itowns, debug, dat, renderer */
+
+// eslint-disable-next-line no-unused-vars
+function showPointcloud(serverUrl, fileName, lopocsTable) {
+    var pointcloud;
+    var oldPostUpdate;
+    var viewerDiv;
+    var debugGui;
+    var view;
+    var positionOnGlobe;
+
+    viewerDiv = document.getElementById('viewerDiv');
+    viewerDiv.style.display = 'block';
+
+    itowns.THREE.Object3D.DefaultUp.set(0, 0, 1);
+
+    itowns.proj4.defs('EPSG:3946',
+        '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 ' +
+        '+y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+    debugGui = new dat.GUI();
+
+    positionOnGlobe = { longitude: 4.631512, latitude: 43.675626, altitude: 250 };
+
+    // TODO: do we really need to disable logarithmicDepthBuffer ?
+    view = new itowns.GlobeView(viewerDiv, positionOnGlobe, {
+        renderer: renderer,
+        handleCollision: false });
+    // view.mainLoop.gfxEngine.renderer.setClearColor(0xcccccc);
+
+    view.controls.minDistance = 0;
+    function addLayerCb(layer) {
+        return view.addLayer(layer);
+    }
+
+    // Configure Point Cloud layer
+    pointcloud = new itowns.GeometryLayer('pointcloud', view.scene);
+    pointcloud.file = fileName || 'infos/sources';
+    pointcloud.protocol = 'potreeconverter';
+    pointcloud.url = serverUrl;
+    pointcloud.table = lopocsTable;
+
+    // point selection on double-click
+    function dblClickHandler(event) {
```

Comment:
Something is off with the formula here, I'm not moving at all or moving off the target position.

Reply:
My bad, it's not supposed to do what I thought